.travis.yml

```diff
@@ -38,7 +38,7 @@ before_install:
 install:
     - conda install --yes python=$PYTHON_BUILD
     - git clone https://github.com/Ouranosinc/pavics-sdi
-    - conda-env update --file environment.yml environment-dev.yml
+    - conda env update --file environment.yml --file environment-dev.yml
```

Comment:
That doesn't work... I've tried it. Now trying conda-merge

Reply:
Really? It works for me locally. What version of `conda` are you using?

Reply:
4.7.10. For me, it only installed the packages from environment-dev.myl